src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -64,13 +68,10 @@
     public ResponseEntity<String> getRequests(Model model, HttpServletRequest request) {
         String eiRequestUrl = eiRequestsControllerUtils.getEIRequestURL(request);
         HttpGet eiRequest = new HttpGet(eiRequestUrl);
-        String header = request.getHeader("Authorization");
 
-        if (header != null) {
-            eiRequest.addHeader("Authorization", header);
-        }
+        eiRequest = (HttpGet) addHeadersToRequest(eiRequest, request);
 
-        return getResponse(eiRequest);
+        return getResponse(eiRequest, request);
```

Comment:
Why two different two requests?
Don't add headers to the existing request?

Reply:
Same comment on the other places in the code where this function is called.

Reply:
Fixed.